README.md

```diff
@@ -7,7 +7,7 @@ This package generate a cross-compiler based on [gcc](https://gcc.gnu.org/) and
 by other systems. You can be able to configure the target of the cross-compiler
 by setting some optional build flags or environment variables:
 
-- **-b**/**BITS**: select between x86 or x86_64
+- **-b**/**BITS**: select between 32 or 64
```

Comment:
Good catch, more clear now. Could you be able to add a note about this only have effect on x86 and x86_64 architectures?